checks/testdata/github-workflow-permissions-run-package-workflow-write.yaml

```diff
@@ -22,5 +22,4 @@ jobs:
       packages: write
     steps:
       - name: some name
-        run: echo "write-and-read workflow"
```

Comment:
Removed this line because parsing this file caused an error. A step is not allowed to have both a `run` and a `uses` field.

Reply:
Nit suggestion: this is a good corner case to test for. We should probably have a test which tests that we handle these parsing issues gracefully.